src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/QueryExecutor.java

```diff
@@ -268,6 +269,7 @@ protected Uni<Optional<TableMetadata>> getSchema(
                   "The provided namespace does not exist: " + namespace));
     }
     // else get the table
+    // TODO: this should probably use CqlIdentifier.fromCql() if we want to be case sensitive
```

Comment:
agree, this can be modified.

Reply:
Leave it as TODO, since it is a easy change but has large impact.

Reply:
Should be follow-up.